src/tab-group.ts

```diff
@@ -408,7 +409,6 @@ export default class TabGroup extends LitElement {
 
   #onTabSelected(event: Event) {
     if (event.target instanceof TabGroupTab) {
-      event.target.selected = true;
```

Comment:
The tab already sets selected when clicked or on key down below, so no need to set it here yet again.